Core/src/org/sleuthkit/autopsy/contentviewers/MediaFileViewer.java

```diff
@@ -120,14 +120,11 @@ public void setFile(AbstractFile file) {
             }
 
             lastFile = file;
-
-            final Dimension dims = MediaFileViewer.this.getSize();
```

Comment:
removing this unnecessary parameter makes the import for java.awt.Dimension also unnecessary.